docs/kepler-model-validation.md

```diff
@@ -70,3 +70,4 @@ Don't edit the following table manually. It is generated by the CI.
 |  2024-08-06  |  v0.7.11-67-gc99e3993  | [Report](validation/2024-08-06/report-v0.7.11-67-gc99e3993.md) | [Artifacts](validation/2024-08-06/validator-v0.7.11-67-gc99e3993) |
 |  2024-08-06  |  v0.7.11-70-gcdcaf00b  | [Report](validation/2024-08-06/report-v0.7.11-70-gcdcaf00b.md) | [Artifacts](validation/2024-08-06/validator-v0.7.11-70-gcdcaf00b) |
 |  2024-08-06  |  v0.7.11-72-g859d9bf1  | [Report](validation/2024-08-06/report-v0.7.11-72-g859d9bf1.md) | [Artifacts](validation/2024-08-06/validator-v0.7.11-72-g859d9bf1) |
+|  2024-08-07  |  v0.7.11-75-g39ef7195  | [Report](validation/2024-08-07/report-v0.7.11-75-g39ef7195-extended.md) | [Artifacts](validation/2024-08-07/validator-v0.7.11-75-g39ef7195-extended) |
```

Comment:
Lets generate this in CI

Reply:
Yes I was just performing a test and it pushed. Future it will just be in CI. I can remove this however.